galata/test/jupyterlab/low-vision.test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, galata, test } from '@jupyterlab/galata';
+
+test.use({
+  autoGoto: false,
+  mockState: galata.DEFAULT_DOCUMENTATION_STATE,
+  viewport: { height: 1024, width: 1280 }
+});
+
+test.describe('Low Vision Support Test', () => {
+  test('Visability of menu-bar at 400-zoom', async ({ page }) => {
+    await page.goto();
+    await page.evaluate('document.body.style.zoom=4.0');
+
+    const imageName = 'visability-of-menu-bar-at-400-zoom.png';
+    expect(
+      await page.screenshot({ clip: { x: 0, y: 0, width: 1280, height: 100 } })
```

Comment:
Why are we only snapshot testing the menu bar?

Reply:
I have done this initially to make sure that the 400% zoom function does what it's meant to and to also create the correct snapshot.

Reply:
Will add more snapshots such as sidebar etc when i get the snapshot working for the menu bar

Reply:
Yes but why not snapshot the whole page?

Reply:
Oops we both commented at the same time. Adding more snapshots later makes sense, thanks!

Reply:
Great no problem!